package.json

```diff
@@ -57,8 +64,8 @@
     "premanifest": "cp -R src/js/ dist/",
     "manifest": "npx @custom-elements-manifest/analyzer analyze --config scripts/custom-elements-manifest.config.js",
     "build:types": "tsc",
-    "build:esm": "esbuild \"src/js/**/*.js\" --target=es2019 --format=esm --outdir=dist",
-    "build:cjs": "esbuild \"src/js/**/*.js\" --target=es2019 --format=cjs --outdir=dist/cjs",
+    "build:esm": "esbuild \"src/js/**/*.js\" src/js/react/**/* --target=es2019 --format=esm --outdir=dist",
+    "build:cjs": "esbuild \"src/js/**/*.js\" src/js/react/**/* --target=es2019 --format=cjs --outdir=dist/cjs",
```

Comment:
esbuild added support for globs recently so it shouldn't be needed

Reply:
Happy to refactor to whatever you're proposing here, though I'm not sure what the concern is?

Reply:
`\"src/js/**/*.js\" src/js/react/**/*` could be `\"src/js/**/*\"